test/unit/account_info/test_sqlite_account_info.py

```diff
@@ -67,7 +67,11 @@ def test_migrate_to_4(self):
 
 class TestSqliteAccountProfileFileLocation:
     @pytest.fixture(autouse=True)
-    def setup(self, monkeypatch, fs):
+    def setup(self, monkeypatch, tmpdir):
+        monkeypatch.setenv(
```

Comment:
I assume that this change allows us to drop `pyfakefs` dependency?

Reply:
yup, it does :)